package.json

```diff
@@ -98,21 +99,21 @@
     "markdown-link-check": "3.11.2",
     "mocha": "10.2.0",
     "named-html-entities-json": "1.0.0",
-    "prettier": "3.0.3",
     "svg-path-bbox": "1.2.5",
     "svg-path-segments": "1.0.0",
     "svglint": "2.4.0",
     "svgo": "3.0.2",
     "svgpath": "2.6.0",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "xo": "0.58.0"
   },
   "scripts": {
     "build": "node scripts/build/package.js",
     "clean": "del-cli index.js index.mjs index.d.ts sdk.js",
-    "format": "prettier --cache --write .",
+    "format": "prettier --cache --write '**/*.md' '**/*.json'",
```

Comment:
We can't remove the prettier dependency if is defined here.

Reply:
Also, why these globs? Why there have been so many removes of the *.prettierignore* file?

Reply:
Can we add a negative glob here to format all files except `.m?js` ones with Prettier?

Reply:
> Can we add a negative glob here to format all files except .m?js ones with Prettier?

Not sure. I will give a try.

Reply:
Prettier does not support `!**/*` pattern.

Reply:
Yes, Prettier uses fast-glob, which uses micromatch that [accepts negation patterns](https://github.com/micromatch/micromatch?tab=readme-ov-file#extglobs). Use `'**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts|svg)'`

Reply:
Note that you don't have added `.json5`, which is the extension of the file *.github/renovate.json5*. I don't want to remember to add more extensions here, a false positive reformat would catch it in the future if we add more files with other extensions.

Reply:
Tried `'**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts|svg)'` but this will include all other files like `.npmrc`, `.gitignore`, etc.

Should I add them to the `.prettierignore`?

Reply:
Yes, please. Anyways XO doesn't formats them.

Reply:
Oh no. I can't add those patterns to `.prettierignore`, and the XO would ignore those files as well.

I just added an `--ignore-unknown` to bypass those unknown files.